misk/src/main/kotlin/misk/client/TypedHttpClientModule.kt

```diff
@@ -61,21 +62,24 @@ class TypedHttpClientModule<T : Any>(
     @Inject
     private lateinit var moshi: Moshi
 
+    @Inject(optional=true)
+    private val tracer: Tracer? = null
+
     override fun get(): T {
       val okhttp = httpClientProvider.get()
 
       val retrofit = Retrofit.Builder()
           .baseUrl(httpClientsConfig[name].url)
           .addConverterFactory(MoshiConverterFactory.create(moshi))
-          .build()
```

Comment:
Why this change?

Reply:
Blerg, merge conflict resolution gone bad. Will fix.